### ISSUE TITLE
Vertical Stepper?

### ISSUE THREAD

**Issue by kamronbatman** [mu29/react-stepper#11]
HI @mu29, do you have a vertical version of this component?

Thanks!

**Comment by daviscabral** [comment]
@kamronbatman have you find anything?

**Comment by kamronbatman** [comment]
I have not. :(

**Comment by daviscabral** [comment]
I was able to hack something on my own - let me know if you want to see.

**Comment by kamronbatman** [comment]
Yes definitely, please link me.  If you have a demo, that is even better!
